modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -242,7 +242,11 @@ const DatePicker = ({
 					}
 
 					if (moment(value).isValid()) {
-						onChange(moment(value).locale(locale).format('L'));
+						onChange(
+							moment(value, getLocaleDateFormat(locale)).format(
+								'MM/DD/YYYY'
+							)
```

Comment:
Hey @rodrigopaulino , Does it mean that it will be always on English format? if the locale is Portuguese?

Reply:
Hi, @jeyvison. Actually, some places (like [this](https://github.com/liferay-data-engine/liferay-portal/blob/01e517e663c4064b6b5981d890b7d27d185eb24b/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js#L110-L116)) internally expects the format to be this way, because moment's default locale is en, in order to create the proper date, but this doesn't affect the UI.

Reply:
Got it. Thanks :)